modules/nw-dpu-configuring-operator.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * networking/networking_operators/configuring-dpu-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-dpu-configuring-operator_{context}"]
+= Configuring the DPU Operator 
+
+* Create a `SriovOperatorConfig` custom resource (CR) to deploy the DPU host components:
```

Comment:
`DpuOperatorConfig` not `SriovOperatorConfig` 

Create a `DpuOperatorConfig` custom resource (CR) to deploy the components necessary to manage the host or the dpu:

^ note also that I lave out `DPU host` in the sentence since it depends on which side you run this.